### DISCUSSION TITLE
Event not triggered on pull request label

### DISCUSSION THREAD

**Discussion by NanoSuarezMa** [community/community#49997] [Actions]
### Select Topic Area

Product Feedback

### Body

I have a workflow that adds labels to the PR. And a second workflow that runs when a pull request is labeled.
But the second one is never executed when the type is **labeled**:
```
on:
  pull_request:
    types: [labeled]
```

If I add **opened** or **synchronize** then is executed, but never with labeled.
```
on:
  pull_request:
    types: [opened, synchronize , labeled]
```
I have an echo printing the event action and it is always opened or synchronize.
` - run: echo "🎉 The job was automatically triggered by a ${{ github.event.action }} type."`

I also tested it, manually adding a label to my PR and only then my workflow runs perfectly and I can see the type labeled.

Does anyone have an idea of what could be the problem? How can I trigger a workflow when my PR is labeled by another workflow?

**Comment by Asheboy** [comment]
@NanoSuarezMa Workflows can't trigger other workflows (i.e. if workflow #1 is setting the label, workflow #2 won't get the labeled event).

**Reply by NanoSuarezMa** [reply]
Hi @Asheboy ,
actually is possible... but my problem was in the personal access token.

I was trying to [Trigger a workflow from a workflow](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow) which requires the first workflow (the one that adds a label) to use a Personal Access Token, rather than the default GITHUB_TOKEN. This is to prevent workflows getting into a recursive workflow loop. The document also includes links on how to store secrets safely.
